ingest/rules/curate.smk

```diff
@@ -18,6 +18,8 @@ rule fetch_general_geolocation_rules:
         general_geolocation_rules="data/general-geolocation-rules.tsv",
     params:
         geolocation_rules_url=config["curate"]["geolocation_rules_url"],
+    log:
```

Comment:
+1 for more use of `log` in Snakemake workflows, but the log file is empty unless you redirect stdout/stderr to the file in the shell command. 

See example in [Nextstrain Snakemake guide](https://docs.nextstrain.org/en/latest/reference/snakemake-style-guide.html#log-standard-out-and-error-output-to-log-files-and-the-terminal)

Reply:
:facepalm:

fixed throughout.